Core/Src/stateEstimation.c

```diff
@@ -27,16 +27,19 @@ void stateEstimation_Update(StateInfo* input) {
 	wheels2Body(input->wheelSpeeds, vel);
 
 	kalman_CalculateK();
-	kalman_Update(input->xsensAcc, vel);
+
+	float meanAcc[2] = {0.0f};
+	movingAverage(input->xsensAcc, meanAcc);
```

Comment:
If I understood correctly, here you are modiifying meanAcc, which is then used in the kalman filter. If this is true, can you add a comment as clarification?